pyfolio/timeseries.py

```diff
@@ -512,7 +516,8 @@ def calc_multifactor(returns, factors):
     return results.params
 
 
-def rolling_beta(returns, benchmark_rets, rolling_window=63):
+def rolling_beta(returns, benchmark_rets,
```

Comment:
I have a weird aversion to calling this `benchmark_rets`, because it's not always the benchmark to which I want to compute beta. I understand, though, that this is the 90% use case. Might it make any sense to rename to something like `factor_rets`, and then in the help string say

factor_rets : pd.Seires or pd.DataFrame
Daily noncumulative returns of the factor to which beta is computed. Usually a benchmark such as the market.

I'm not strong on this issue, more a point for discussion because I'm doing all this in the lectures and seeing how academics refer to these concepts.